README.md

```diff
@@ -716,6 +105,10 @@ This table describes the list of available parameters for Helm Chart.
 | `priorityClassName`      | Name of a PriortyClass defined to set pod priority        | `""`                               |
 | `updateStrategy`      | Update strategy for helm chart and app version updates        | `OnDelete`                               |
 
-# Known Issues and Limitations
+## Known Issues and Limitations
 
 1. If the hostname is greater than 64 characters there may be issues with certificates. The certificates may shorten the name or use SANs for hostnames in the certificates.
```

Comment:
"The certificates may shorten the name or use SANs for hostnames in the certificates." Rereading this, I don't think this is clear what the issue is. How would a certificate "shorten the name"? We want users to keep host names less than 64 characters or use certs with SANs?

Reply:
updated limitation